### DISCUSSION TITLE
Candle for Windows with arm64 support

### DISCUSSION THREAD

**Discussion by GeekCornerGH** [wixtoolset/issues#6514] [Questions] [answered]
## Feature requests

If this issue is a feature request:

* Describe the scenario and benefits that the feature supports.
Adding arm64 support would add ability to build arm64 msi with Electron-builder

> (Provide lots of specific detail so we understand the context around the feature.)  

Context : I am trying to build an Electron app for Windows arm64 in msi format with GitHub Actions.

* Describe how you're accomplishing the feature today (if possible).
I can't

> (Be as specific as you can and err on the side of providing too much information.)

* Describe what you'd like the new feature to do.
I want to be able to build arm64 msi with Electron-builder on GitHub Actions.

> (There's no need to design it but suggestions are appreciated.)

**Comment by robmen** [comment, marked as answer]
This is already supported in WiX v4-preview.0

**Comment by rseanhall** [comment]
And v3.14 - https://github.com/wixtoolset/issues/issues/5558

**Comment by GeekCornerGH** [comment]
> And v3.14 - https://github.com/wixtoolset/issues/issues/5558

@rseanhall Ok, so how do I use this with Actions?

**Comment by rseanhall** [comment]
The WiX Toolset does not use GitHub issues for support. I have converted this issue into a discussion item, which is an experiment that we're trying out. Alternatively, you may want to ask your question on the [wix-users mailing list](https://wixtoolset.org/documentation/mailinglist/) or [Stack Overflow](https://stackoverflow.com/questions/tagged/wix).
